src/app/utils/item.js

```diff
@@ -78,8 +106,11 @@ function LibraryItem() {
     // nypl-owned ReCAP
     const nyplRecap = !!(holdingLocation && !_isEmpty(holdingLocation) &&
       holdingLocation['@id'].substring(4, 6) === 'rc');
-    let deliveryLocations = item.deliveryLocation && item.deliveryLocation.length ?
-      item.deliveryLocation : this.getDeliveryLocations(item, true);
+    const barcode = (item.identifier && item.identifier.length) ?
```

Comment:
It would be better if you did something like

```
const identifiersArray = [{ name: 'barcode', value: 'urn:barcode:' }];
const bibIdentifiers = this.getIdentifiers(item.identifier, identifiersArray);
const barcode = bibIdentifiers.barcode || '';
```

That way, if there are more identifiers, you can pull them from the `bibIdentifiers` object instead of calling that function multiple times.